pkg/systemd/terminal.jsx

```diff
@@ -40,8 +40,23 @@ const _ = cockpit.gettext;
 
         constructor(props) {
             super(props);
-            const theme = document.cookie.replace(/(?:(?:^|.*;\s*)theme_cookie\s*=\s*([^;]*).*$)|^.*$/, "$1");
-            const size = document.cookie.replace(/(?:(?:^|.*;\s*)size_cookie\s*=\s*([^;]*).*$)|^.*$/, "$1");
+
+            let theme = cockpit.localStorage.getItem('terminal-theme');
+            let size = cockpit.localStorage.getItem('terminal-font-size');
+            // HACK: Try to read the configuration from localStorage, if it does not exists fall back
+            // to the old configuration stored in a browser's cookie. After enough time has been
+            // passed this can be dropped.
+            if (!theme) {
+                theme = document.cookie.replace(/(?:(?:^|.*;\s*)theme_cookie\s*=\s*([^;]*).*$)|^.*$/, "$1");
+                cockpit.localStorage.setItem('terminal-theme', theme);
+                this.setCookie("theme_cookie", theme);
```

Comment:
This is confusing to read now. Maybe rename to `invalidateCookie` or so?

Reply:
Sure! I thought I'd clear the unused cookies and save those precious bytes being sent over the wire every HTTP request.